packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -308,31 +319,27 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
             </Stack>
           </Stack>
           <Divider />
-          {!isLoading && !Boolean(error) && (
-            <CloudPulseLineGraph
-              error={
-                status === 'error'
-                  ? error?.[0]?.reason || 'Erorr while rendering widget'
-                  : undefined
-              }
-              legendRows={
-                legendRows && legendRows.length > 0 ? legendRows : undefined
-              }
-              ariaLabel={props.ariaLabel ? props.ariaLabel : ''}
-              data={data}
-              gridSize={widget.size}
-              loading={isLoading}
-              nativeLegend={true}
-              showToday={today}
-              timezone={timezone}
-              title={''}
-              unit={unit}
-            />
-          )}
-          {isLoading && <CircleProgress />}
-          {Boolean(error?.length) && (
-            <ErrorState errorText={error![0].reason} />
-          )}
+
+          <CloudPulseLineGraph
+            error={
+              status === 'error'
+                ? error?.[0]?.reason ?? 'Error while rendering graph'
+                : undefined
+            }
+            legendRows={
+              legendRows && legendRows.length > 0 ? legendRows : undefined
+            }
+            ariaLabel={ariaLabel ? ariaLabel : ''}
```

Comment:
could use a nullish coalescing operator here

```suggestion
            ariaLabel={ariaLabel ?? ''}
```